azuredevops/internal/acceptancetests/testutils/hcl.go

```diff
@@ -200,6 +200,24 @@ resource "azuredevops_serviceendpoint_dockerregistry" "serviceendpoint" {
 	return fmt.Sprintf("%s\n%s", projectResource, serviceEndpointResource)
 }
 
+// HclServiceEndpointAzureCRResource HCL describing an AzDO service endpoint
+func HclServiceEndpointAzureCRResource(projectName string, serviceEndpointName string /*, username string, password string*/) string {
+	serviceEndpointResource := fmt.Sprintf(`
+resource "azuredevops_serviceendpoint_azurecr" "serviceendpoint" {
+	project_id             = azuredevops_project.project.id
+	service_endpoint_name  = "%s"
+	azurecr_spn_tenantid      = "9c59cbe5-2ca1-4516-b303-8968a070edd2"
+    azurecr_subscription_id   = "3b0fee91-c36d-4d70-b1e9-fc4b9d608c3d"
```

Comment:
Nit: format this hcl section (a way to do this is by putting this into a terraform file and formatting that file with terraform fmt) and can you add one tab to lines 206-215, so that it would be on the same level with the method?